wdfn-server/waterdata/templates/macros/components.html

```diff
@@ -2,8 +2,12 @@
     <div class="wdfn-component" data-component="hydrograph" data-siteno="{{ site_data.site_no }}"
          data-agency-cd="{{ site_data.agency_cd }}" data-sitename="{{ site_data.station_nm }}"
          data-parameter-code="{{ default_parameter_code }}">
-        <div class="loading-indicator-container"></div>
-        <div class="graph-container"></div>
+        <div id="hydrograph-date-controls-container"></div>
```

Comment:
It was a lot easier to create containers for the different parts of the hydrograph component rather than rely on order of rendering. I think this will allow us to play a little more with layout too.